browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -3,15 +3,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "brave/browser/net/brave_network_delegate_browsertest.h"
```

Comment:
no header file is needed for the tests, everything can be done in the cc file which is the convention

Reply:
also I don't understand the purpose of this change, the indentation change from adding the header file makes it extremely difficult to see what actually changed in the the test

Reply:
We have added the header so that we can subclass the `BraveNetworkDelegateBrowsertest` fixture. This allows us to avoid a pretty significant amount of code reuse.